### ISSUE TITLE
Validate local action exists

### ISSUE THREAD

**Issue by TWiStErRob** [TWiStErRob/net.twisterrob.ghlint#242]
uses: ./.github/actions/track-workflow-result

**Comment by timsearle** [comment]
Happy to give this a go

**Comment by TWiStErRob** [comment]
@timsearle Go ahead.

I guess let's start with just raw checks, and then later maybe add checkout validation: e.g. checkout from another repo should disable the check? Checkout into a specific directory from current repo could be handled. Do as much or as little as you want.

**Comment by TWiStErRob** [comment]
Handle this somehow?

```yaml
uses: org/repo/actions/action-name
```
which is missing the version.

If cannot be detected in scope of this issue, open a new one before merging the PR.
